src/panels/lovelace/cards/hui-gauge-card.ts

```diff
@@ -128,24 +128,20 @@ class HuiGaugeCard extends LitElement implements LovelaceCard {
 
     const name = this._config.name ?? computeStateName(stateObj);
 
-    const hasAnyAction =
-      !this._config.tap_action ||
-      hasAction(this._config.tap_action) ||
-      hasAction(this._config.hold_action) ||
-      hasAction(this._config.double_tap_action);
-
     // Use `stateObj.state` as value to keep formatting (e.g trailing zeros)
     // for consistent value display across gauge, entity, entity-row, etc.
     return html`
       <ha-card
-        class=${classMap({ action: hasAnyAction })}
+        class=${classMap({ action: hasAnyAction(this._config) })}
         @action=${this._handleAction}
         .actionHandler=${actionHandler({
           hasHold: hasAction(this._config.hold_action),
           hasDoubleClick: hasAction(this._config.double_tap_action),
         })}
         tabindex=${ifDefined(
-          hasAction(this._config.tap_action) ? "0" : undefined
+          !this._config.tap_action || hasAction(this._config.tap_action)
```

Comment:
was missing the tab stop for the case where there's no defined tap action, which defaults to more-info